Makefile

```diff
@@ -31,3 +31,8 @@ server:
 .PHONY: clean
 clean:
 	yarn run clean
+
+.PHONY: test_unit_codecov
+test_unit_codecov: test_unit
+	npm install codecov -g
+	codecov -f .coverage/coverage-final.json
```

Comment:
This doesn't appear to actually run the unit tests anymore, just generates code coverage. I think we need something like `yarn test && yarn run codecov`.

Reply:
Nvmd, I missed that it has a dependency on the other make target